tests/integration_tests/charts/commands_tests.py

```diff
@@ -85,6 +86,49 @@ def test_export_chart_command(self, mock_g):
             "version": "1.0.0",
         }
 
+    @patch("superset.security.manager.g")
+    @pytest.mark.usefixtures("load_energy_table_with_slice")
+    def test_export_chart_with_query_context(self, mock_g):
```

Comment:
Can you rename this to `test_import_...` and move it to `TestImportChartsCommand` instead?

Reply:
Hey, of course. Though shouldn't it be with the rest of the export tests?

Reply:
Oh, my bad.... I read the docstring below and got confused because it says "Test that charts that have a query_context are imported correctly". Sorry about that!